File: kwadratowa.py
import math

a = input("set a ->")
b = input("set b ->")
c = input("set c ->")

def kwadratowa(a, b, c):
    x1 = 0.0
    x2 = 0.0
    if a == 0:
        return [null,null,null],False
    delta = pow(b,2) - (4*a*c)
    if delta >= 0:
        x1 = (-1 * b - math.sqrt(delta))/ (2 * a)
        x2 = (-1 * b + math.sqrt(delta))/ (2 * a)
        positiveDelta = True
    else:
        positiveDelta = False
    return [delta,x1,x2],positiveDelta

r,positiveDelta = kwadratowa(float(a),float(b),float(c))

if positiveDelta:
    print("x1 = ",r[1])
    print("x2 = ",r[2])
print("Δ = ",r[0])

File: kwadratowa.java
import java.util.Scanner;

public class kwadratowa{

    static boolean positiveDelta = false;

    public static void main(String[] args){
        float a,b,c;
        Scanner scanner = new Scanner(System.in);
        do{
            System.out.print("Set a ->");
            a = scanner.nextFloat();
        }while(a == 0);
        System.out.print("Set b ->");
        b = scanner.nextFloat();
        System.out.print("Set c ->");
        c = scanner.nextFloat();
        scanner.close();

        float[] r = kwadratowa(a, b, c);

        if(positiveDelta){
            System.out.printf("x1 = %f\n",r[1]);
            System.out.printf("x2 = %f\n",r[2]);
        }
        System.out.printf("delta = %f\n",r[0]);
    }

    static float[] kwadratowa(float a, float b, float c){
        float[] r = new float[3];
        float delta = (float)Math.pow(b,2) + (-4*a*c);
        r[0] = delta;
        if(0 <= delta){
            r[1] = (float)(-1*b - Math.sqrt(delta));
            r[2] = (float)(-1*b + Math.sqrt(delta));
            positiveDelta = true;
        }else positiveDelta = false;
        return r;
    }
}